doc/conf.py

```diff
@@ -45,25 +45,26 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Hdlparse'
-copyright = u'2017, Kevin Thibedeau'
-author = u'Kevin Thibedeau'
+project = 'Hdlparse'
```

Comment:
The small edits here will conflict with `umarcor/doc-btd`, but I'm not sure the status of that PR. I'd ideally merge that before doing clean-up here. It seems our intention is to change the name to pyHDLParser since I don't think anyone has heard anything from Kevin.